Simplenote/Classes/SPSettingsViewController.m

```diff
@@ -660,11 +660,33 @@ - (void)touchIdSwitchDidChangeValue:(UISwitch *)sender
 {
     SPPinLockManager.shared.shouldUseBiometry = sender.on;
 
-    if (![self isPinLockEnabled]) {
-        [self showPinLockViewController];
+    if (![self isPinLockEnabled] && [self.biometrySwitch isOn]) {
+        UIAlertController* alert = [self pinLockRequiredAlert];
+        [self presentViewController:alert animated:YES completion:nil];
     }
 }
 
+- (UIAlertController*)pinLockRequiredAlert
+{
+    NSString* alertTitle = NSLocalizedString(([NSString stringWithFormat:@"To enable %@, you must have a passcode setup first.", self.biometryTitle]),
+                                             @"Prompt to setup passcode before biomtery lock");
```

Comment:
Note: this localization string won't be correctly exported because localization "key" is known only in runtime.
Basically it is now like this:
```
NSString *localizationKey = can-be-anything;
NSString *alertTitle = NSLocalizedString(localizationKey, @"...");
```

When we try to export localization keys to get them translated, the tool won't know what is the key. It is a variable, so it can be anything.

Localization keys have to be explicit. In this particular case we can rewrite it like this:

```
NSString *alertTitleTemplate = NSLocalizedString(@"To enable %1$@, you must have a passcode setup first.", @"Prompt to setup passcode before biomtery lock. Paramteres: %1$@ - biometry type (Face ID or Touch ID)");
NSString *alertTitle = [NSString stringWithFormat: alertTitleTemplate, self.biometryTitle];
```